actors.md

```diff
@@ -186,16 +189,16 @@ Parameters:
 Return: Address
 
 ```go
-func CreateStorageMiner(worker Address, sectorSize BytesAmount, pid PeerID) Address {
```

Comment:
this change is incorrect. please keep it as an address.

Reply:
Roger that. I inferred pubkey from its use at 198, and `storage-market.md`. Address is consistent with the constructor.